wandb/apis/public/api.py

```diff
@@ -630,13 +630,23 @@ def _parse_artifact_path(self, path):
         entity = self.settings["entity"] or self.default_entity
         if path is None:
             return entity, project
+
+        path_alias = path.split(":")
+        if len(path_alias) == 1:
+            alias = ""
+        elif len(path_alias) == 2:
```

Comment:
its possible for `len(path_alias) > 2`
example:
<img width="725" alt="Screenshot 2024-05-20 at 15 45 15" src="https://github.com/wandb/wandb/assets/8826594/d721b38b-3426-4a25-8344-7de2266e407d">
In this case `len(path_alias) == 5`. I think doing `> 1` works here